assets/README.md

```diff
@@ -38,6 +42,10 @@ for details.
 
 Build the JavaScript in `js` into a bundled file using [webpack].
 
+### `epub-javascript`
```

Comment:
Should we normalize all names to be `FORMATTER-TASK`? So we would have `epub-javascript` and `html-javascript`. Otherwise I am afraid it will be confusing. What do you think?